caffe2/CMakeLists.txt

```diff
@@ -1443,6 +1458,9 @@ if(BUILD_MOBILE_TEST)
   endforeach()
 endif()
 
+# add_executable(lite_interpreter_demo ${TORCH_ROOT}/test/cpp/demos/main.cpp)
```

Comment:
Is it replaced by unittest somewhere?

Reply:
The unittest is added in the latest commit. lite_interpreter_demo is probably not needed and removed.